src/components/Button/styles.module.scss

```
@@ -6,6 +6,7 @@
     border: none;
     cursor: pointer;
     font-weight: bolder;
+    font-size: 25px;
     transition: all 300ms;
     &:hover {
         background: white;
```

Replace this color for #FFF; on the _colors.scss file

```suggestion
        background: $white;
```